Rakefile.rb

```diff
@@ -26,23 +27,25 @@ def json_write(file, json)
   # we only build any country that contains that string. For example:
   #    EP_COUNTRY_REFRESH=Latvia be rake countries.json
   to_build = ENV['EP_COUNTRY_REFRESH'] || 'data'
-
-  countries = @HOUSES.group_by { |h| h.split('/')[1] }.select do |_, hs|
-    hs.any? { |h| h.include? to_build }
+  countries = EveryPolitician.countries.select do |c|
+    c.slug.downcase.include? to_build.downcase
```

Comment:
That's the current behaviour, so removing it would be a big change, especially as it's behaviour that I often take advantage of from the command line (e.g. for countries that I can't always remember how to spell!) 

The goal of this argument is really about speed, not isolation, and the only penalty here is in saving a little _less_ time over the full rebuild than an exact match would do. 

In the case where this is triggered by a bot, and we can rely on an exact match, it might make sense to have a way to say so. But the extra value we'd gain from that doesn't really seem worth the extra overhead of having two arguments or a different way to flag that up. Definitely open to suggestions of how we could have both behaviours without any extra complexity, though!

Printing out the countries it's changing is tempting, though again it would be a change of behaviour if it suddenly had output by default, so we'd probably want to hide that behind a verbose flag or something, and again I'm not sure it's important enough. Open to persuasion though if you want to open a ticket for it.